pymongo/compression_support.py

```diff
@@ -123,16 +122,10 @@ class ZlibContext(object):
     compressor_id = 2
 
     def __init__(self, level):
-        self.compress: Callable[[bytes], bytes]
```

Comment:
Why as this code removed as opposed to moving it into `def compress` below? Is this logic no longer necessary for Jython compatibility?

Reply:
We don't support Jython.

Reply:
Ah, I see your comment below addresses part of this. I'm still curious why the Jython workarounds are no longer necessary, did we drop support for Jython?